app/CommandExecutor.php

```diff
@@ -83,19 +102,50 @@ public function execute(string $command, string $cwd) : int
         return proc_close($proc);
     }
 
+    public function getOutputBuffer(): string
+    {
+        if (! $this->outputFileName) {
```

Comment:
there will always be a file name defined now, right?

Reply:
tempnam function returns false in case of failure, that's why we're checking it